src/setup.js

```diff
@@ -137,7 +137,7 @@ function configure({ env, stage, stageUrl, apiStage, localhostUrl, checkoutUri,
     }
 
     if (localhostUrl) {
-        config.localhostUrl = localhostUrl;
+        // localhostUrl is not configurable and hardcoded to localhost.paypal.com
```

Comment:
leaving config signature intact and passing on the setter here. i could be convinced to fully remove it as well, but I was hoping this would leave more historical information for the next person.